include/mbgl/util/constants.hpp

```diff
@@ -43,6 +43,8 @@ constexpr Seconds CLOCK_SKEW_RETRY_TIMEOUT { 30 };
 
 constexpr UnitBezier DEFAULT_TRANSITION_EASE = { 0, 0, 0.25, 1 };
 
+constexpr const char* API_BASE_URL = "https://api.mapbox.com/";
```

Comment:
Let's remove the trailing slash from this URL. That matches GL JS and is more correct -- a trailing slash is a path, and in this case the point is that this is a base URL to which a path will be appended.